chrome/content/zotero/preferences/preferences_export.js

```diff
@@ -133,35 +139,55 @@ Zotero_Preferences.Export = {
 	},
 	
 	
-	updateQuickCopyHTMLCheckbox: function () {
+	updateQuickCopyUI: function () {
 		var format = document.getElementById('zotero-quickCopy-menu').value;
+		
 		var mode, contentType;
 		
-		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		[mode, format] = format.split('=');
 		[mode, contentType] = mode.split('/');
 		
+		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		checkbox.checked = contentType == 'html';
 		checkbox.disabled = mode != 'bibliography';
+		
+		var menulist = document.getElementById('locale-menu');
+		if (mode != 'bibliography') {
+			// maintain menu selection but show blank label
+			menulist.insertItemAt(0, '', menulist.value);
+			menulist.selectedIndex = 0;
+			menulist.disabled = true;
```

Comment:
Also fixed for the site-specific settings window: https://github.com/rmzelle/zotero/commit/691649600fa016cbf362160da23f14eec7ee221e
(forget my comments about `<treecol/>`; I was looking at the wrong file and confused)